src/core/ddsc/include/dds/ddsc/dds_public_status.h

```diff
@@ -482,7 +482,7 @@ dds_get_sample_lost_status (
  */
 DDS_EXPORT dds_return_t
 dds_get_requested_deadline_missed_status (
-  dds_entity_t reader,
```

Comment:
This really only takes a reader, so I think "reader" is the better choice.

Reply:
Applied and fixed.